### PR TITLE
College assessment dash

### PR DESCRIPTION
**Pull request by cbini**
# Pull Request

## Merge Pre-Check

- [x] Create a Pull Request task on our
      [Data Warehouse](https://app.asana.com/0/35737884613515/1204991342481434) Asana project
- [x] <kbd>Format</kbd> has been run on all requested files
- [x] `dbt build` successfully run for the entire project[^1]

[^1]:
    If only requesting 1 file, it's acceptable to run build only for that and its downstream
    dependencies.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205640748104471
  - https://app.asana.com/0/0/1205971968562508
  - https://app.asana.com/0/0/1205972182005154
  - https://app.asana.com/0/0/1206107176497529

### CODE REVIEW

File: src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+                when a.subject_area = 'Mathematics' then 'Math' else a.subject_area
+            end as subject_area,
+            a.date_taken as test_date,
+            a.response_type,  -- Group or overall
+            a.response_type_description,  -- Group name
+            a.points,  -- Points earned... looks to be # of questions correct on Illuminate
+            round(a.percent_correct / 100, 2) as percent_correct,  -- % correct field on Illuminate
+
+            count(distinct a.subject_area) over (
+                partition by
```
Comment (anthonygwalters) [inline src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql:25]: @GabyRangelB this is cool & also smart

File: src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+            ) as scale_score,
+
+        from {{ ref("int_assessments__response_rollup") }} as a
+        inner join
+            {{ ref("stg_assessments__act_scale_score_key") }} as ssk
+            on a.assessment_id = ssk.assessment_id
+            and a.points between ssk.raw_score_low and ssk.raw_score_high
+        where
+            a.scope in ('ACT', 'SAT')
+            and a.academic_year = {{ var("current_academic_year") }}
```
Comment (anthonygwalters) [inline src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql:56]: @GabyRangelB this is also cool I like this too

**Review by anthonygwalters — CHANGES_REQUESTED** [review]
@GabyRangelB cool cool cool

File: src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql (hunk 1 above)
Comment (anthonygwalters) [reply src/dbt/kipptaf/models/assessments/intermediate/int_assessments__college_assessment_practice.sql:25]: test

**Comment by cbini** [comment]
asdf

**Pull request merged**